src/store/mutation-types.js

```diff
@@ -37,6 +37,10 @@ export const SET_DISPLAY_PRICE_RANGE = 'SET_DISPLAY_PRICE_RANGE'
 export const SET_SEARCH_DATES = 'SET_SEARCH_DATES'
 export const SET_SEARCH_LOCATION = 'SET_SEARCH_LOCATION'
 export const UNSET_SEARCH_LOCATION = 'UNSET_SEARCH_LOCATION'
+export const SET_MAP_CENTER = 'SET_MAP_CENTER'
+export const UNSET_MAP_CENTER = 'UNSET_MAP_CENTER'
+export const SET_MAX_DISTANCE = 'SET_MAX_DISTANCE'
+export const SEARCH__USE_MAP_CENTER = 'SEARCH__USE_MAP_CENTER'
```

Comment:
You can merge two of these at least (see my comment in mutations file).

I would personally merge the four of them.